src/atomic.rs

```diff
@@ -887,14 +900,14 @@ impl<'g, T> Shared<'g, T> {
     ///
     /// Dereferencing a pointer is unsafe because it could be pointing to invalid memory.
     ///
-    /// Another concern is the possiblity of data races due to lack of proper synchronization.
+    /// Another concern is the possiblity of word races due to lack of proper synchronization.
```

Comment:
Search and replace? ;)

Reply:
Haha, you have a good eye!

Reply:
I would stick to `data` though, since `word` is generally used when the specific width of the object matters, and is quite ambiguous. In particular, x86 intel assembly refers to words as being 2 bytes rather than the machine word (4 on x86, 8 on x86_64) size.

Reply:
Hm, I didn't know that. Okay, I'm reverting back to `data`. Maybe we'll come up with a better name next time...